developer-hub/src/polkadot_sdk/polkadot.rs

```diff
@@ -0,0 +1,58 @@
+//! # Polkadot
+//!
+//! Implementation of the Polkadot host in Rust.
+//!
+//! ## Getting Involved
+//!
+//! - [Polkadot Forum](https://forum.polkadot.network/)
+//! - Polkadot Fellowship
+//! 	- [Runtimes](https://github.com/polkadot-fellows/runtimes)
+//! 	- [RFCs](https://github.com/polkadot-fellows/rfcs)
+//! - [Polkadot Specs](spec.polkadot.network)
+//! - [The Polkadot Parachain Host Implementers' Guide](https://paritytech.github.io/polkadot-sdk/book/)
+//! - [Whitepaper](https://www.polkadot.network/whitepaper/)
+//!
+//! ## Platform
+//!
+//! ### Polkadot 1.x
+//!
+//! The original vision of Polkadot (i.e. **Polkadot 1**) revolves around the following arguments:
+//!
+//! * Future is multi-chain, because we need different chains with different specialization to
+//!   achieve widespread goals.
+//! * In other words, no single chain is good enough to achieve this.
+//! * A multi-chain future will inadvertently suffer from fragmentation of economic security.
+//!   * This stake fragmentation will make communication over consensus system with varying security
+//!     levels inherently unsafe.
+//!
+//! Polkadot's answer to the above is:
+//!
+//! * Shared Security: The idea of shared economic security sits at the core of Polkadot. Polkadot
+//!   enables different blockchains (ie. "*Parachains*") to pool their economic security from
+//!   Polkadot (ie. "*Relay Chain*").
+//! * A framework to build blockchains: In order to materialize the multi-chain future, an easy
+//!   blockchain framework must exist. This is [`crate::polkadot_sdk::substrate`],
+//!   [`crate::polkadot_sdk::frame_runtime`] and [`crate::polkadot_sdk::cumulus`].
+//! * A communication language between blockchains: In order for these blockchains to communicate,
+//!   they need a shared language. [`crate::polkadot_sdk::xcm`] is one such language.
+//!
+//! > Note that the interoperability promised by Polkadot is unparalleled in that any two parachains
+//! > connected to Polkadot have the same security and can have much higher guarantees about the
+//! > security of the recipient of any message.
+//!
+//! Polkadot delivers the above vision, alongside a flexible means for parachains to schedule
+//! themselves with the Relay Chain. To achieve this, Polkadot has been developed with an
+//! architecture similar to that of a computer. Polkadot Relay Chain has a number of "cores". Each
+//! is (in simple terms) is capable of progressing 1 parachain a a time. For example, a parachain
+//! can schedule itself for on a single core for 5 blocks.
+//!
+//! Within the scope of Polkadot 1.x, two main scheduling ways has been considered:
+//!
+//! * Long term Parachains, obtained through locking a sum of DOT in an auction system.
+//! * on-demand Parachains, purchased through paying DOT to the relay-chain whenever needed.
+//!
+//! This scheduling system, and its evolution is the segway into Polkadot 2.x
+//!
+//! ### Polkadot 2.x
```

Comment:
fyi, it's possible that we can take a lot of inspiration about what's in the Wiki for here in the future.  although this may be "too high level", need to probably tailor this to how its useful to a programmer: https://wiki.polkadot.network/docs/polkadot-direction

Reply:
Glossing over the wiki, it already looks pretty good. It is slightly verbose, but that is fine for a wiki. I think it is great that you are finding these similarities and making sure information is consistent. 

Admittedly, I don't know the wiki too well. Given that this is a small resource, it should be easy for you and Radha to really know the content here a-z as it is being made.

Reply:
On this, we can probably create a version which caters to what a developer need to know, then link both the wiki and RFCs